modules/virt-virtctl-commands.adoc

```diff
@@ -103,27 +103,29 @@ You use `virtctl` connection commands to expose ports and connect to VMs and VMI
 |`virtctl console <vm_name>`
 |Connect to the serial console of a VM.
 
-|`virtctl expose <vm_name>`
+|`virtctl expose vm <vm_name> --name <service_name> --type <ClusterIP\|NodePort\|LoadBalancer> --port <port>`
 |Create a service that forwards a designated port of a VM and expose the service on the specified port of the node.
 
+Example: `virtctl expose vm rhel9_vm --name rhel9-ssh --type NodePort --port 22`
+
 |`virtctl scp -i <ssh_key> <file_name> <user_name>@<vm_name>`
-|Copy a file from your machine to a VM. This command uses the private key of an SSH key pair. The VM must be configured with the public key.
+|Copy a file from your machine to a VM. This command uses the private key of an SSH key pair. The VM must be configured with the public key before it boots.
```

Comment:
The key can be configured in other ways too. It doesn't have to be before it boots.